packages/utils/platform-tools/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: ['@fluentui-react-native/eslint-config-rules'],
+  rules: {
+    '@typescript-eslint/no-var-requires': 0,
```

Comment:
Why is turning this rule off required for the whole package? If needed I would rather this be localized to the instance it's needed for instead of for everything

Reply:
Removed the rule.